bin/Powheg/runcmsgrid_powheg.sh

```diff
@@ -373,8 +373,8 @@ if [ -s pwg-stat.dat ]; then
   tail -${tail} cmsgrid_final.lhe                           >  cmsgrid_final.lhe_tail
   head -${head} cmsgrid_final.lhe                           >  cmsgrid_final.lhe_F
   proclin=`expr $head + 1`
-  PROCESS=`sed -n -e ${proclin}p  cmsgrid_final.lhe |  awk '{print $4}'`
-  echo "  "$XSECTION"   "$XSECUNC"  1.00000000000E-00 "$PROCESS >>  cmsgrid_final.lhe_F
+  proc=`sed -n -e ${proclin}p  cmsgrid_final.lhe |  awk '{print $4}'`
```

Comment:
ok, so I think I uderstand (?) more or less what could be happening, and should be related to this PR
https://github.com/cms-sw/genproductions/pull/1064 that was done to fix an hard-coded value for the process number (?) https://github.com/yuanchao/genproductions/commit/835e5c0ce66828e7e4803f323ee6c3d53fee809b#diff-8cf10f8734b1bb4d2bcecd3878fdbc28L389
causing null xsec in genXsecAnalyzer. Thinking again about it, I am surprised that genXsecAnalyzer uses information put by hand in the lhe file.

So, as you say when the word "PROCESS" was used to name the variable, it was overwritten with the process name. I am not sure why when this variable name contains an underscore it should complain.

May I ask to do a check:
- produce a lhe file with this fix using ggH_quark-mass-effect, and post it somewhere for us to inspect
- convert the lhe file to edm 
- use GenXsecAnalyzer to check that a non-null value is returned

Thanks a lot
Luca